tiledb/api/examples/quickstart_dense.rs

```diff
@@ -18,6 +19,12 @@ fn array_exists() -> bool {
     )
 }
 
+/// Creates a dense array at URI `QUICKSTART_DENSE_ARRAY_URI`.
```

Comment:
Something to think about here but a goal of the test suite in core is to try and move to a place where we can run all of these things against different URIs to prove out our VFS implementations. One nice thing about TileDB is that just changing an array URI from `file:///some/path/to/array_name` to `s3://bucketname/array_name` makes things Just Work &trade;

I'm absolutely not saying we should do this work as part of this PR, but given that we're just starting our test suite from scratch we should probably keep this use case top of mind.

Reply:
🙇